jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java

```diff
@@ -100,18 +83,15 @@
     DataSetClientServices dataSetClientServices;
 
     @Inject
-    public FilterEditorPopup(DataSetClientServices dataSetClientServices,
-                             DataSetFilterEditor filterEditor) {
-        this.filterEditor = filterEditor;
-        this.dataSetClientServices = dataSetClientServices;
-        setBody(uiBinder.createAndBindUi(FilterEditorPopup.this));
+    private ManagedInstance<DataSetFilterEditor> filterEditorsProvider;
```

Comment:
Wondering if we need to use manage instance in here, isn't enough changing the scope to ApplicationScoped?